features/step_definitions/operator.rb

```diff
@@ -9,3 +9,141 @@
   })
   step %Q/the step should succeed/
 end
+
+Given /^etcd operator "([^"]*)" is removed successfully from "([^"]*)" project$/ do | name, proj_name|
+  ensure_admin_tagged
+  raise "!!! Error doesn't exist etcd operator: #{name} in this #{proj_name} project" unless subscription("#{name}").exists?
+  step %Q/I run the :delete client command with:/, table(%{
+    | object_type       | subscription |
+    | object_name_or_id | #{name}      |
+    | n                 | #{proj_name} |
+  })
+  raise "Error removing subscription: #{name} in #{proj_name} project" unless @result[:success]
+
+  step %Q/I run the :delete client command with:/, table(%{
+    | object_type       | clusterserviceversion |
+    | object_name_or_id | etcdoperator.v0.9.4   |
```

Comment:
hardcoding version?

Reply:
Yes, there is no good way to find the exact CSV resource since the label are random.
```
$ oc get csv etcdoperator.v0.9.4 -o yaml
...
  labels:
    olm.api.2c1e6f7e17c07035: provided
    olm.api.2fdc3540750c4d2b: provided
    olm.api.c571d720f17289d3: provided
```
I will changed it to `--all`. Maybe it's a better choice.